client/scss/elements/_root.scss

```diff
@@ -2,6 +2,8 @@
     @include define-color('color-primary', #007d7e);
     @include define-color('color-primary-darker', css-darken(css-adjust-hue(get-color('color-primary'), 1), 4%));
     @include define-color('color-primary-dark', css-darken(css-adjust-hue(get-color('color-primary'), 1), 7%));
+    @include define-color('color-primary-lighter', css-lighten(css-adjust-hue(get-color('color-primary'), 1), 10%));
```

Comment:
Are these colours from the design system?

Reply:
Not yet - we have an additional Draftail styling ticket for tweaking the colours/design, these are just "roughly similar" placeholders.